app/views/scap_contents/_form.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= javascript 'foreman_openscap/scap_edit_content' %>
```

Comment:
Using `javascript` from layout helper breaks turbolinks benefits, because it adds  js file to head, which creates  unnecessary roundtrip.